storybook/src/components/Row/Row.stories.tsx

```diff
@@ -3,43 +3,47 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { Avatar, Button, Heading, Link, Paragraph, Row } from '@amsterdam/design-system-react/src'
+import { Avatar, Heading, Link, Row } from '@amsterdam/design-system-react/src'
 import { crossAlignOptions, mainAlignOptions } from '@amsterdam/design-system-react/src/common/layout'
 import { Meta, StoryObj } from '@storybook/react'
 
-const ThreeButtons = Array.from(Array(3).keys()).map((i) => <Button key={i}>Button {i + 1}</Button>)
-const ThreeParagraphs = [
-  <Paragraph className="ams-docs-pink-box" key={1} style={{ inlineSize: '8rem' }}>
-    One line
-  </Paragraph>,
-  <Paragraph className="ams-docs-pink-box" key={2} style={{ inlineSize: '8rem' }}>
-    Two
-    <br />
-    lines
-  </Paragraph>,
-  <Paragraph className="ams-docs-pink-box" key={3} style={{ inlineSize: '8rem' }}>
-    One line
-  </Paragraph>,
+const ThreeItems = [
+  <div className="ams-docs-item" key={0} />,
+  <div className="ams-docs-item" key={1} />,
+  <div className="ams-docs-item" key={2} />,
 ]
 
 const meta = {
   title: 'Components/Layout/Row',
   component: Row,
   args: {
-    children: ThreeButtons,
+    children: ThreeItems,
+    className: 'ams-docs-row',
   },
   argTypes: {
     align: {
-      control: 'radio',
-      options: mainAlignOptions,
+      control: {
+        type: 'radio',
+        labels: { undefined: 'start' },
+      },
+      options: [undefined, ...mainAlignOptions],
     },
     alignVertical: {
-      control: 'radio',
-      options: crossAlignOptions,
+      control: {
+        type: 'radio',
+        labels: { undefined: 'stretch' },
+      },
+      options: [undefined, ...crossAlignOptions.filter((option) => option !== 'baseline')],
```

Comment:
Why is `baseline` filtered from the options here? You can't recreate the 'Align opposing texts'-story this way.

Reply:
Fixed – may have mixed up Row and Column halfway.